sdk/keyvault/Azure.Security.KeyVault.Secrets/tests/compatibility/ExpectedAotWarnings.txt

```diff
@@ -0,0 +1,2 @@
+.*/Azure\.Security\.KeyVault\.Secrets.src.SecretClientBuilderExtensions\.cs\(30\): Trim analysis warning IL2026: Microsoft\.Extensions\.Azure\.SecretClientBuilderExtensions\.AddSecretClient<TBuilder,TConfiguration>\(TBuilder,!!1\): Using member 'Azure\.Core\.Extensions\.IAzureClientFactoryBuilderWithConfiguration`1<TConfiguration>\.RegisterClientFactory<SecretClient,SecretClientOptions>\(TConfiguration\)' which has 'RequiresUnreferencedCodeAttribute' can break functionality when trimming application code\. Binding strongly typed objects to configuration values is not supported with trimming\. Use the Configuration Binder Source Generator \(EnableConfigurationBindingGenerator=true\) instead\.
```

Comment:
TODO - clean these up in https://github.com/Azure/azure-sdk-for-net/issues/44296